### PR TITLE
feat: Execute the version validation in a separate script (SDKCF-6040)

### PR DESCRIPTION
**Pull request by malayli**

### CODE REVIEW

File: scripts/version-validation.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+then
+    if [[ $1 =~ ^(0|[1-9][[:digit:]]*)\.(0|[1-9][[:digit:]]*)\.(0|[1-9][[:digit:]]*)$ ]];
+    then
+        echo "version input is valid";
+    else
+        echo "version input is invalid";
+        exit 1;
+    fi
+else
+    echo "RELEASE_VERSION build parameter is missing";
```
Comment (imaginaris) [inline scripts/version-validation.sh]: `RELEASE_VERSION` var is out of this context - it's Bitrise specific. It's better to print something more generic like "version input data is missing"

File: scripts/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+- Click on `Add Environment Variable`
+
+- Disable `Replace variables in input`
+
+- Click on `Start Build`
+
+## Example
+- Command line:
+```bash
+sh version-validation.sh 10.0.0
```
Comment (imaginaris) [inline scripts/README.md:42]: 10.0.0 or $RELEASE_VERSION ?

Comment (malayli) [reply scripts/README.md:42]: @imaginaris It's an example, then it is `10.0.0`.

File: scripts/version-validation.sh (hunk 1 above)
Comment (malayli) [reply scripts/version-validation.sh]: @imaginaris Yes, I forgot to remove it. I changed the string. Fixed.

**Review by imaginaris — APPROVED** [review]

**Pull request merged**